.github/workflows/test.yml

```diff
@@ -41,18 +41,19 @@ jobs:
       with:
         lane: 'scan'
 
-    # Read generate script file to learn more about this CI step. 
-    - name: Generate code coverage report from .xcresult/
-      run: ./scripts/generate-code-coverage-report.sh 
+    # https://about.codecov.io/blog/pre-converting-xcresult-files-for-codecov-using-xcresultparser/
+    - name: Generate code coverage report from .xcresult/ that can be converted to codecov format
+      run: |
+        brew tap a7ex/homebrew-formulae && brew install xcresultparser
+        xcresultparser --output-format cobertura "Customer.io-Package.xcresult" > ".build/coverage.xml"
```

Comment:
This is the most important part of this PR. The previous tool that we were using to convert `xcresult` file into a format that codecov understands was no longer working. So, I went out and tried to find a popular tool that we can depend on into the future. 

Because codecov made a blog post that included this tool, it made me feel confident in this tool.